core/trino-main/src/main/java/io/trino/operator/scalar/MapToMapCast.java

```diff
@@ -235,15 +235,17 @@ public static Block checkBlockIsNotNull(Block value)
     public static Block mapCast(
             MethodHandle keyProcessFunction,
             MethodHandle valueProcessFunction,
-            Type toMapType,
+            Type toType,
```

Comment:
targetType, destinationType ?

Reply:
`targetType` works, or I could also drop this commit.

This one was because when I was reading the function, I kept getting confused and thinking "what's being converted to a map type?"